docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -894,30 +894,41 @@ This event will be sent when there are failures in RTMP streaming and/or browser
 3. Incorrect storage (S3) credentials
 4. Other unexpected errors
 
+Please check the below table to check various error types, messages and description.
```

Comment:
@chdinesh1089 Let's reword this:

Please check the below table for possible error types, messages and description.